FrameworkDesignConsiderations.md

```diff
@@ -28,7 +28,7 @@ follow the same pattern: read data and then transform it new data, and then make
 data available in the TES. As such, one could consider writing one single generic algorithm
 which wraps this functionality (and thus provides a uniform pattern), and delegate the
 computational work. This suggests a new type of Gaudi component, which is explictly provided
-with input data, and explicitly provides output data -- lets define this as a Transform.. 
+with input data, and explicitly provides output data -- let's define this as a Transform.
 Note that there is no direct interaction between these Transform components and the TES -- this
```

Comment:
I like this idea very much, but it makes me wonder about the role of the TES in this architecture: jsut as a container of objects to persist ?